esp_port/docs/camera_device.puml

```diff
@@ -0,0 +1,92 @@
+@startuml
```

Comment:
I saw this which you might want to look at -- to get these displayed in one of the readmes: https://github.com/Zingam/UML-in-Markdown

Alternatively, GitHub supports mermaid diagrams - https://github.blog/developer-skills/github/include-diagrams-markdown-files-mermaid/